docs/package_management.rst

```diff
@@ -0,0 +1,56 @@
+.. _package_management:
+
+Package Management
+==================
+
+.. py:module:: web3.pm
+
+Web3.py provides a number of features for installing and using resources in
+EthPM packages.  These features enable a user to perform any task or realize
+any use-case detailed in the `EthPM specification
+<http://ethpm.github.io/ethpm-spec/index.html>`_.  Below, we'll go over a
+number of examples of how to use this functionality in web3.py.
+
+.. _pm_download_a_package:
+
+Downloading a Package
+---------------------
+
+To use a package with web3.py, it must be downloaded or read from the local
+filesystem.  In the code snippet below, a number of different approaches are
+used to get a package object for a ``greeter`` package:
+
+.. code-block:: python
+
+    >>> from web3 import Web3, HTTPProvider
+    >>> w3 = Web3(HTTPProvider('https://ropsten.infura.io/<token>'))
+
+    >>> # Get a package from a registry
+    >>> greeter_pkg = w3.pm.get_package_from_registry('greeter', registry='ens://packages.ethereum.eth')
```

Comment:
It's not necessary - but should we add a version param and a note that it's optional (or if left out will return the most recent version)?